docker/docker-compose.yml

```diff
@@ -169,17 +169,16 @@ services:
       timeout: 3s
       retries: 3
     volumes:
-      - dbdata:/var/lib/postgresql/data
+      - ${DB_DATA_VOLUME_PREVIOUS:-dbdata_pg13}:/var/lib/postgresql/data
     ports:
       - 5432
     expose:
       - 5432
 
-  # This should no longer need to be overridden for ARM
-  # processors
+  # This should no longer need to be overridden for ARM processors
   db:
```

Comment:
I think an `image` value will force a build and allow switching branches. Maybe "image: warehousepg:16.0".  My initial boot on this branch created and initialized the pg16 volume with pg13.